### DISCUSSION TITLE
Is it possible to create content with status as "Live" and skip the review process using the /content/v2/create API?

### DISCUSSION THREAD

**Discussion by vijaykhollam** [Sunbird-Knowlg/Community#231] [Q&A] [answered]
I am trying to use the /content/v2/create API to create content with a "Live" status directly. However, when I pass status: "Live" in the request payload, I receive the following error: "Properties in list [status, dialcodes] are not allowed in request"

**API Request:**

Here’s the request payload I am using:
```
{
    "request": {
        "content": {
            "name": "Sample Content",
            "code": "sample-content-123456789",
            "mimeType": "application/pdf",
            "primaryCategory": "Learning Resource",
            "status": "Live"
        }
    }
}
```

API Documentation Reference:

I am referring to the documentation for the /content/v2/create API:
[Content Create API Documentation](https://documenter.getpostman.com/view/25463377/2s9YC4VDL4#30136156-f3c5-48dc-b1ec-407be72111c5)

**Questions:**
1. Is it possible to set the content status to "Live" during creation?
2. Does the API support a parameter like skipReview or any alternative way to bypass the review process?

Any help or insights would be appreciated. Thank you!

@snehal0904 @siddhi0719 @komalm @vinod303

**Comment by pallakartheekreddy** [comment, marked as answer]
@vijaykhollam 
Thank you for your question!
1. The `/content/v2/create` API does not allow the status field to be included in the request payload. The status is managed by the system based on the content lifecycle, and it cannot be set to `Live` directly during content creation.
2. Currently, there is no option to skip or bypass the review process using the API. The review and approval workflow is integral to the content publishing process to ensure quality and compliance with standards.

**Reply by vijaykhollam** [reply]
@pallakartheekreddy Thank you for the clarification!
